drivers/pwm/pwm_nrfx.c

```diff
@@ -121,6 +121,63 @@ static int pwm_nrfx_init(struct device *dev)
 	return 0;
 }
 
+#ifdef CONFIG_DEVICE_POWER_MANAGEMENT
+
+static int pwm_nrfx_set_power_state(u32_t new_state, struct device *dev)
+{
+	if (new_state == DEVICE_PM_ACTIVE_STATE) {
+		int res = pwm_nrfx_init(dev);
+
+		assert(res == 0);
+	} else {
+		assert(new_state == DEVICE_PM_LOW_POWER_STATE ||
+		       new_state == DEVICE_PM_SUSPEND_STATE ||
+		       new_state == DEVICE_PM_OFF_STATE);
+		const struct pwm_nrfx_config *pconfig =
+						dev->config->config_info;
```

Comment:
One last thing. Please, correct this indentation to:
```
		const struct pwm_nrfx_config *pconfig =
			dev->config->config_info;
```
and this PR will be ready for merging.